tests/unit/Types/ContextFactoryTest.php

```diff
@@ -13,12 +13,12 @@
 namespace phpDocumentor\Reflection\Types {
 
 // Added imports on purpose as mock for the unit tests, please do not remove.
+    use \ReflectionClass;
     use Mockery as m;
-    use phpDocumentor\Reflection\DocBlock,
```

Comment:
These imports are part of the unittest. please add them back

Reply:
They're still in there... it looks like ECS just reorders them.